python/pyspark/serializers.py

```diff
@@ -427,6 +427,64 @@ def load_stream(self, stream):
                     'Invalid number of pandas.DataFrames in group {0}'.format(dataframes_in_group))
 
 
+class ArrowStreamArrowUDFSerializer(ArrowStreamSerializer):
```

Comment:
Also, I don't think we should add this special serializer for each pandas UDF but rather we should just have a switch to each existing serializers so that it works for all types of pandas UDFs.